### ISSUE TITLE
[Bug] Custom DOM fails with interaction layer (globalZIndex: 1)

### ISSUE THREAD

**Issue by xiaoluoHe** [VisActor/VChart#3587]
### Version

1.13.1

### Link to Minimal Reproduction

none

### Steps to Reproduce
```js
const spec = {
  type: 'area',
  data: {
    values: [
      { month: 'Jan', country: 'Africa', value: 4229 },
      { month: 'Jan', country: 'EU', value: 4376 },
      { month: 'Jan', country: 'China', value: 3054 },
      { month: 'Jan', country: 'USA', value: 12814 },
      { month: 'Feb', country: 'Africa', value: 3932 },
      { month: 'Feb', country: 'EU', value: 3987 },
      { month: 'Feb', country: 'China', value: 5067 },
      { month: 'Feb', country: 'USA', value: 13012 },
      { month: 'Mar', country: 'Africa', value: 5221 },
      { month: 'Mar', country: 'EU', value: 3574 },
      { month: 'Mar', country: 'China', value: 7004 },
      { month: 'Mar', country: 'USA', value: 11624 },
      { month: 'Apr', country: 'Africa', value: 9256 },
      { month: 'Apr', country: 'EU', value: 4376 },
      { month: 'Apr', country: 'China', value: 9054 },
      { month: 'Apr', country: 'USA', value: 8814 },
      { month: 'May', country: 'Africa', value: 3308 },
      { month: 'May', country: 'EU', value: 4572 },
      { month: 'May', country: 'China', value: 12043 },
      { month: 'May', country: 'USA', value: 12998 },
      { month: 'Jun', country: 'Africa', value: 5432 },
      { month: 'Jun', country: 'EU', value: 3417 },
      { month: 'Jun', country: 'China', value: 15067 },
      { month: 'Jun', country: 'USA', value: 12321 },
      { month: 'Jul', country: 'Africa', value: 13701 },
      { month: 'Jul', country: 'EU', value: 5231 },
      { month: 'Jul', country: 'China', value: 10119 },
      { month: 'Jul', country: 'USA', value: 10342 },
      { month: 'Aug', country: 'Africa', value: 4008, forecast: true },
      { month: 'Aug', country: 'EU', value: 4572, forecast: true },
      { month: 'Aug', country: 'China', value: 12043, forecast: true },
      { month: 'Aug', country: 'USA', value: 22998, forecast: true },
      { month: 'Sept', country: 'Africa', value: 18712, forecast: true },
      { month: 'Sept', country: 'EU', value: 6134, forecast: true },
      { month: 'Sept', country: 'China', value: 10419, forecast: true },
      { month: 'Sept', country: 'USA', value: 11261, forecast: true }
    ]
  },
  stack: true,
  xField: 'month',
  yField: 'value',
  seriesField: 'country',
  axes: [
    {
      type: 'band',
      orient: 'bottom',
      label: {
        style: (value) => ({
          fill:false,
          html: {
            visible: true,
            style: {
              pointerEvents: 'auto',
              color: '#87898B',
              fontSize: '12px',
              fontStyle: 'normal',
              fontWeight: 400,
              lineHeight: '16px',
              letterSpacing: '0.161px',
            },
            dom: `<div>${value}HTML!!!</div>`,
          },
        }),
        
      }
    },
    {
      type: 'linear',
      orient: 'left',
      grid: {
        visible: true,
        style: {
          lineDash: [3, 3],
        },
      },
    },
  ],
  point: {
    style: {
      // globalZIndex: 1,
      size: 0
    },
    state: {
      dimension_hover: {
        globalZIndex: 1,
        size: 10,
        outerBorder: {
          distance: 0,
          lineWidth: 6,
          strokeOpacity: 0.2
        }
      }
    }
  },
  line: {
    style: {
      // Configure the lineDash attribute based on the forecast field value of the data
      lineDash: data => {
        if (data.forecast) {
          return [5, 5];
        }
        return [0];
      }
    }
  },
  area: {
    style: {
      fillOpacity: 1,
      textureColor: '#fff',
      textureSize: 14,
      // Configure the texture attribute based on the forecast field value of the data
      texture: data => {
        if (data.forecast) {
          return 'bias-rl';
        }
        return null;
      }
    }
  },
  legends: [{ visible: true, position: 'middle', orient: 'bottom' }],
  crosshair: {
    gridZIndex: 450,
    xField: {
      visible: true,
      line: {
        type: 'line',

      }
    }
  }
};

const vchart = new VChart(spec, { dom: CONTAINER_ID, enableHtmlAttribute: true });
vchart.renderSync();

// Just for the convenience of console debugging, DO NOT COPY!
window['vchart'] = vchart;
```js
### Current Behavior

bottom 轴的 html 标签消失
<img width="843" alt="image" src="https://github.com/user-attachments/assets/efe54587-2a73-4b50-9def-3d15d9bb3f1f" />


### Expected Behavior

正常显示轴标签
<img width="801" alt="image" src="https://github.com/user-attachments/assets/010bc21f-bba9-40b2-93c6-a662b0904d3a" />


### Environment

```markdown
- OS:
- Browser:
- Framework:
```

**Comment by neuqzxy** [comment]
closed by https://github.com/VisActor/VRender/pull/1626

**Comment by xile611** [comment]
fixed in 1.13.2
